app/tests/algorithms_tests/test_forms.py

```diff
@@ -661,3 +665,77 @@ def test_model_version_control_form():
     assert "Model updating already in progress." in str(
         form.errors["algorithm_model"]
     )
+
+
+@pytest.mark.django_db
+class TestJobCreateForm(TestCase):
```

Comment:
Indeed, there shouldn't be a need to introduce new `TestCase` classes.

Reply:
Just for my understanding: why is TestCase not good to use?

Reply:
It's from a test framework that we don't use. We use pytest instead.